app/components/pages/SubmissionWizard/steps/Editors/EditorsContent.js

```diff
@@ -79,6 +86,45 @@ class EditorsContent extends React.Component {
     )
   }
 
+  handleSuggestedReviewersChanged = event => {
+    this.props.handleChange(event)
+
+    const MAXIMUM = limits.suggestedReviewers.max
```

Comment:
MAXIMUM? More intuitive name like `maxReviewers`? The original name is also intuitive (`limits.suggestedReviewers.max`).

Reply:
done - thanks